packetbeat/publish/publish.go

```diff
@@ -84,6 +84,7 @@ func (p *TransactionPublisher) CreateReporter(
 
 	// load and register the module it's fields, tags and processors settings
 	meta := struct {
+		Index      string                  `config:"index"`
```

Comment:
The new `index` options should also go into the asciidoc and reference.yml configs.